analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BreakOutsideSwitch.CSharp7.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Tests.Diagnostics
```

Comment:
For the same reason stated above - this can be added to the already existing test file.